.github/workflows/nodejs.condo.cypress.yml

```diff
@@ -46,13 +46,11 @@ jobs:
       - name: run build
         run: |
           set -x
-          echo '# Build time .env config!' > .env
-          echo 'COOKIE_SECRET=undefined' >> .env
-          echo 'DATABASE_URL=${{ matrix.database }}' >> .env
-          echo 'WORKER_REDIS_URL=undefined' >> .env
-          echo 'FAKE_ADDRESS_SUGGESTIONS=true' >> .env
-          echo 'NODE_ENV=development' >> .env
-          export NODE_ENV=development
+          echo "# Build time .env config!" > .env
+          echo "COOKIE_SECRET=undefined" >> .env
+          echo "DATABASE_URL=undefined" >> .env
+          echo "REDIS_URL=undefined" >> .env
```

Comment:
I guess it's should be redundant too, since we've removed vars from dev-portal....So only node_env is acceptable

Reply:
not yet, its hart to remove it at the moment